.circleci/main-config.yml

```diff
@@ -128,6 +128,9 @@ commands:
             DDP-?(*))
               DEPLOY_ENV=dev
             ;;
+            PEPPER-?(*))
+              DEPLOY_ENV=dev
+            ;;
             ?(*_)test)
               DEPLOY_ENV=test
             ;;
```

Comment:
General comment: I kept what I thought have to be fixed in this file but removed my DSM secret changes

Reply:
I'm following the pattern here. I noticed that branches labeled `DDP-*` can be deployed to the dev environment. However, it seems that the `DDP-` naming convention was used before we transitioned to being the Pepper swarm and updated our Jira ticket naming from `DDP-` to `PEPPER-*`. Considering this change in naming, I'm concerned that we might have lost some functionality during this transition. To address this, I went ahead and included the `PEPPER-*` branches for deployment as well. This is to ensure that we're not missing any relevant changes that were made after the naming switch.

Reply:
Thanks for the explanation. Let me see if I understand...
This script supports deployment of our RC branches? And those are typically titled like `dsm_rc_swarm_sprint_12`, although hotfix branches likely have different names. But since that branch name does not appear to match any of these cases, my mental model must be wrong. What am I missing?

Also, how did we choose a new test executor image? It looks like that image was built in March. Do we know why it was not put into use then?
`